lib/functions/SpecialFunctions.ts

```diff
@@ -251,7 +251,11 @@ async function inRecursiveAsync(
   }
 
   try {
-    const next = await evaluate(args.shift(), mapping);
+    const nextExpression = args.shift();
+    if (!nextExpression) {
+      throw new Error('Create to catch');
```

Comment:
This error message is a bit unclear (also the next one)

Reply:
There's a try catch. I'm fairly certain that when there's no nextExpression we should jump to the catch. This error would never see the surface. Maybe I should write that in the message?

Reply:
I would advise against that, as it makes the logic flow harder to follow. Perhaps you can just `return` there instead?

Reply:
That should be possible yes.

Reply:
Wow, just realize we don't need any of that. We check the length so we could just use a cast.